tests/js/spec/views/alerts/create.spec.jsx

```diff
@@ -161,140 +171,119 @@ describe('ProjectAlertsCreate', function () {
-      );
-
-      wrapper
-        .find('RuleNode')
-        .at(0)
-        .find('button[aria-label="Delete Node"]')
-        .simulate('click');
+      await selectEvent.select(getByText('Add optional condition...'), [
+        'A new issue is created',
+      ]);
+      fireEvent.click(getByLabelText('Delete Node'));
 
       // Add another condition
-      selectByValue(
-        wrapper,
-        'sentry.rules.conditions.tagged_event.TaggedEventCondition',
-        {selector: 'Select[placeholder="Add optional condition..."]'}
-      );
+      await selectEvent.select(getByText('Add optional condition...'), [
+        "An event's tags match {key} {match} {value}",
+      ]);
 
       // Edit new Condition
-      const ruleNode = wrapper.find('RuleNode').at(0);
+      const ruleNode = getByText("An event's tags match").parentElement;
 
-      ruleNode
-        .find('input[name="key"]')
-        .simulate('change', {target: {value: 'conditionKey'}});
-
-      ruleNode
-        .find('input[name="value"]')
-        .simulate('change', {target: {value: 'conditionValue'}});
+      fireEvent.change(ruleNode.querySelector('input[name="key"]'), {
+        target: {value: 'conditionKey'},
+      });
+      fireEvent.change(ruleNode.querySelector('input[name="value"]'), {
+        target: {value: 'conditionValue'},
+      });
 
-      selectByValue(wrapper, 'ne', {name: 'match', control: true});
+      await selectEvent.select(getByText('equals'), ['does not equal']);
 
       // Add a filter and remove it
-      selectByValue(wrapper, 'sentry.rules.filters.age_comparison.AgeComparisonFilter', {
-        selector: 'Select[placeholder="Add optional filter..."]',
-      });
-
-      wrapper
-        .find('RuleNode')
-        .at(1)
-        .find('button[aria-label="Delete Node"]')
-        .simulate('click');
+      await selectEvent.select(getByText('Add optional filter...'), [
+        'The issue is {comparison_type} than {value} {time}',
+      ]);
+      fireEvent.click((await findAllByLabelText('Delete Node'))[1]);
 
       // Add a new filter
-      selectByValue(wrapper, 'sentry.rules.filters.age_comparison.AgeComparisonFilter', {
-        selector: 'Select[placeholder="Add optional filter..."]',
-      });
-
-      const filterRuleNode = wrapper.find('RuleNode').at(1);
+      await selectEvent.select(getByText('Add optional filter...'), [
+        'The issue is {comparison_type} than {value} {time}',
+      ]);
 
-      filterRuleNode
-        .find('input[type="number"]')
-        .simulate('change', {target: {value: '12'}});
+      const filterRuleNode = getByText('The issue is').parentElement;
+      // console.log(debug(filterRuleNode));
 
-      // Add an action and remove it
-      selectByValue(wrapper, 'sentry.rules.actions.notify_event.NotifyEventAction', {
-        selector: 'Select[placeholder="Add action..."]',
+      fireEvent.change(filterRuleNode.querySelector('input[type="number"]'), {
```

Comment:
can we maybe query the selector by its name or aria-label? 

here is one example: 
https://github.com/getsentry/sentry/blob/faa7208c747ed49077e827dc4efaa6c1bddc0f99/tests/js/spec/views/issueList/tagFilter.spec.tsx#L53

Reply:
```suggestion
      fireEvent.change(filterRuleNode.querySelector('input[type="number"]'), {
```